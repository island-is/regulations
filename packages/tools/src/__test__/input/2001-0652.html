<div align="center">
  1. gr.
  </div>

  <p>Eftirfarandi reglugerðir ráðsins og framkvæmdastjórnarinnar, sem vísað er til í liðum 15h, 15j og 15k í XIII. kafla II. viðauka við samninginn um Evrópska efnahagssvæðið, eins og honum var breytt með ákvörðun sameiginlegu EES-nefndarinnar nr. 75/2000 frá 2. október 2000 um breytingu á II. viðauka (Tæknilegar reglugerðir, staðlar, prófanir og vottun) við EES-samninginn, skulu öðlast gildi hér á landi með þeim breytingum og viðbótum sem leiðir af II. viðauka samningsins, bókun 1 við samninginn og öðrum ákvæðum hans:<br />
    1. <a href="/media/attachments/398R2743.doc">Reglugerð ráðsins (EB) nr. 2743/98 frá 14. desember 1998</a> um breytingu á reglugerð (EB) nr. 297/95 um gjöld sem greiða ber til Lyfjamálastofnunar Evrópu.<br />
    2. <a href="/media/attachments/398R1146.doc">Reglugerð framkvæmdastjórnarinnar (EB) nr. 1146/98</a> frá 2. júní 1998 um breytingu á reglugerð (EB) nr. 541/95 um umfjöllun um breytingar á skilmálum markaðsleyfis sem lögbært yfirvald aðildarríkis hefur veitt.<br />
    3. <a href="/media/attachments/398R1069.doc">Reglugerð framkvæmdastjórnarinnar (EB) nr. 1069/98</a> frá 26. maí 1998 um breytingu á reglugerð (EB) nr. 542/95 frá 10. mars 1995 um umfjöllun um breytingar á skilmálum markaðsleyfis sem fellur undir gildissvið reglugerðar ráðsins (EBE) nr. 2309/93.<br />
  <br />
  <br />
  </p>

  <div align="center">
  2. gr.
  </div>

  <p>Reglugerðirnar og <a href="/media/attachments/akv 75 2000.doc">ákvörðun sameiginlegu EES-nefndarinnar nr. 75/2000</a>, sbr. 1. gr., sem birt hefur verið í EES-viðbæti við Stjórnartíðindi EB nr. 59, 14. desember 2000, eru birtar sem fylgiskjal með reglugerð þessari. Tilskipanir framkvæmdastjórnarinnar sem getið er um í ákvörðun sameiginlegu EES-nefndarinnar nr. 75/2000 hafa þegar verið innleiddar með lyfjalögum nr. 93/1994, með síðari breytingum, og reglugerðum settum með stoð í þeim.<br />
  <br />
  <br />
  </p>

  <div align="center">
  3. gr.
  </div>

  <p>Brot gegn reglugerð þessari varða fangelsi eða sektum, nema þyngri refsing liggi við samkvæmt öðrum lögum.<br />
  <br />
  <br />
  </p>

  <div align="center">
  4. gr.
  </div>

  <p>Reglugerð þessi, sem sett er með stoð í 44. gr. lyfjalaga nr. 93/1994, með síðari breytingum, öðlast þegar gildi.<br />
  <br />
  <br />
  </p>

  <div align="center">
  <em>Heilbrigðis- og tryggingamálaráðuneytinu, 22. ágúst 2001.</em>
  </div>
  <br />


  <div align="center">
  <strong>Jón Kristjánsson.</strong>
  </div>

  <div align="right">
  <em>Davíð Á. Gunnarsson.</em>
  </div>

