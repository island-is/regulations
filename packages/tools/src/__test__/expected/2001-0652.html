<h3 class="article__title">
1.
gr.
</h3>

<p>
Eftirfarandi
reglugerðir
ráðsins
og
framkvæmdastjórnarinnar,
sem
vísað
er
til
í
liðum
15h,
15j
og
15k
í
XIII.
kafla
II.
viðauka
við
samninginn
um
Evrópska
efnahagssvæðið,
eins
og
honum
var
breytt
með
ákvörðun
sameiginlegu
EES-nefndarinnar
nr.
75/2000
frá
2.
október
2000
um
breytingu
á
II.
viðauka
(Tæknilegar
reglugerðir,
staðlar,
prófanir
og
vottun)
við
EES-samninginn,
skulu
öðlast
gildi
hér
á
landi
með
þeim
breytingum
og
viðbótum
sem
leiðir
af
II.
viðauka
samningsins,
bókun
1
við
samninginn
og
öðrum
ákvæðum
hans:<br />
1.
<a href="https://files.reglugerd.is/media/attachments/398R2743.doc">Reglugerð
ráðsins
(EB)
nr.
2743/98
frá
14.
desember
1998</a>
um
breytingu
á
reglugerð
(EB)
nr.
297/95
um
gjöld
sem
greiða
ber
til
Lyfjamálastofnunar
Evrópu.<br />
2.
<a href="https://files.reglugerd.is/media/attachments/398R1146.doc">Reglugerð
framkvæmdastjórnarinnar
(EB)
nr.
1146/98</a>
frá
2.
júní
1998
um
breytingu
á
reglugerð
(EB)
nr.
541/95
um
umfjöllun
um
breytingar
á
skilmálum
markaðsleyfis
sem
lögbært
yfirvald
aðildarríkis
hefur
veitt.<br />
3.
<a href="https://files.reglugerd.is/media/attachments/398R1069.doc">Reglugerð
framkvæmdastjórnarinnar
(EB)
nr.
1069/98</a>
frá
26.
maí
1998
um
breytingu
á
reglugerð
(EB)
nr.
542/95
frá
10.
mars
1995
um
umfjöllun
um
breytingar
á
skilmálum
markaðsleyfis
sem
fellur
undir
gildissvið
reglugerðar
ráðsins
(EBE)
nr.
2309/93.
</p>

<h3 class="article__title">
2.
gr.
</h3>

<p>
Reglugerðirnar
og
<a href="https://files.reglugerd.is/media/attachments/akv 75 2000.doc">ákvörðun
sameiginlegu
EES-nefndarinnar
nr.
75/2000</a>,
sbr.
1.
gr.,
sem
birt
hefur
verið
í
EES-viðbæti
við
Stjórnartíðindi
EB
nr.
59,
14.
desember
2000,
eru
birtar
sem
fylgiskjal
með
reglugerð
þessari.
Tilskipanir
framkvæmdastjórnarinnar
sem
getið
er
um
í
ákvörðun
sameiginlegu
EES-nefndarinnar
nr.
75/2000
hafa
þegar
verið
innleiddar
með
lyfjalögum
nr.
93/1994,
með
síðari
breytingum,
og
reglugerðum
settum
með
stoð
í
þeim.
</p>

<h3 class="article__title">
3.
gr.
</h3>

<p>
Brot
gegn
reglugerð
þessari
varða
fangelsi
eða
sektum,
nema
þyngri
refsing
liggi
við
samkvæmt
öðrum
lögum.
</p>

<h3 class="article__title">
4.
gr.
</h3>

<p>
Reglugerð
þessi,
sem
sett
er
með
stoð
í
44.
gr.
lyfjalaga
nr.
93/1994,
með
síðari
breytingum,
öðlast
þegar
gildi.
</p>

<p class="Dags" align="center">
<em>Heilbrigðis-
og
tryggingamálaráðuneytinu,
22.
ágúst
2001.</em>
</p>

<p align="center">
<strong>Jón
Kristjánsson.</strong>
</p>

<p align="right">
<em>Davíð
Á.
Gunnarsson.</em>
</p>
